src/VizTab.tsx

```diff
@@ -31,30 +31,70 @@ import { matchTopic } from './util';
 import PoseSetter from './components/PoseSetter';
 import config from './config';
 
-function createPose2DfMsg({
+function createPoseStampedMsg({
   namespace,
+  topic,
+  frame = 'map',
   x,
   y,
   theta,
 }: {
   namespace: string;
+  topic: string;
+  frame: string;
   x: number;
   y: number;
   theta: number;
 }) {
   const fbb = new flatbuffers.Builder();
-  const pose = fb.amrl_msgs.Pose2Df.createPose2Df(
+
+  const metadataOffset = fb.MsgMetadata.createMsgMetadata(
+    fbb,
+    fbb.createString('geometry_msgs/PoseStamped'),
+    fbb.createString(`${namespace}/${topic}`)
+  );
+
+  const frameOffset = fbb.createString(frame);
+
+  fb.std_msgs.Header.startHeader(fbb);
+  fb.std_msgs.Header.addStamp(
```

Comment:
`std_msgs/Header` has a `seq` field that should by incremented every time a new message is sent by a node to the topic.

Reply:
The seq field appears to be behaving correctly, looking at the messages being published on the robot

Reply:
Being tracked here, will followup later, since this affects other messages we send from robofleet as well:
https://github.com/ut-amrl/robofleet_webviz/issues/13